src/ensembl/src/content/app/browser/browser-track-config/BrowserTrackConfig.tsx

```diff
@@ -98,28 +99,41 @@ const BrowserTrackConfig: FunctionComponent<BrowserTrackConfigProps> = (
     props.updateApplyToAll(!applyToAll);
   }, [applyToAll, updateApplyToAll]);
 
+  const checkboxStyles: any = {
+    checked: styles.allTracksChecked,
+    checkboxHolder: styles.allTracks
+  };
+
   return (
     <section className={styles.trackConfig}>
       <dl>
-        <dd className={styles.allTracks}>
-          <input
-            type="checkbox"
-            defaultChecked={applyToAll}
+        <dd>
+          <Checkbox
+            classNames={checkboxStyles}
+            label={'All tracks'}
+            checked={applyToAll}
             onChange={applyToAllToggle}
           />
-          <label htmlFor="">All tracks</label>
         </dd>
       </dl>
       <dl>
         <dd>
-          <label htmlFor="">Track name</label>
-          <button className={styles.trackConfigSlider} onClick={toggleName}>
+          <label htmlFor={'trackConfig-trackName'}>Track name</label>
```

Comment:
The `htmlFor` attribute on a label (which translates into the `for` attribute of the [html `label` tag](https://www.w3schools.com/tags/tag_label.asp) makes sense only if we are binding the label to an element with the corresponding id (usually — to an input). Here we are not; so it's useless.

Frankly, I do not understand this makrup. Why did Imran use the definition list (dl, dd) tags, when it's not a definition list? Why are we using label tags, although there is no corresponding input or select? I would have changed it to just divs or spans. Not necessarily now; perhaps during a later refactor.

Reply:
Regarding label `for`: In our case I added it to turn on/off trackConfigSlider when clicking on label itself, just like any other input. 

dd/dl:  True that it is not a def list. Divs are better in this case. I dont think we need to wait for refactoring. Lets clean it as we move.

Reply:
Changes pushed.